lib/perl/Genome/Disk/Command/Allocation/Move.pm

```diff
@@ -47,23 +47,31 @@ sub execute {
     my $self = shift;
 
     for my $allocation ($self->allocations) {
-        my %params;
-        if ($self->target_volume) {
-            $params{target_mount_path} = $self->target_volume->mount_path;
-        }
+        $allocation->move( $self->_resolve_move_params_for_allocation($allocation) );
+    }
+
+    $self->status_message("Successfully moved allocations!");
+    return 1;
+}
 
+sub _resolve_move_params_for_allocation {
+    my ($self, $allocation) = @_;
+
+    my %params;
+    if ($self->target_volume) {
+        $params{target_mount_path} = $self->target_volume->mount_path;
+    }
+    else { 
```

Comment:
If the user specifies both a `--target-volume` and a `--target-group` in this command should it check to make sure they match before ignoring the latter parameter?  (In this case we do have both the objects!)